Week1/MAKEME.md

```diff
@@ -7,108 +7,219 @@ Topics discussed this week:
 • API calls
 ```
 
-
 >[Here](/Week3/README.md) you find the readings you have to complete before the ninth lecture.
 
 ## Step 1: Feedback
 
-_Deadline Monday_
-
-Please provide feedback in an issue.
-
-_Deadline Monday_
+**_Deadline Monday_**
 
-## Step 2: FINISH ALL YOUR JAVASCRIPT HOMEWORK
+Please provide feedback on last week's homework from a fellow student as a GitHub issue.
 
-_Deadline Saturday_
+## Step 2: Single Page Application :sweat_drops:
 
-:point_up:
+**_Deadline Thursday_**
 
-## Step 3: SPA :sweat_drops:
+_This homework is more extensive and challenging than previous homework! Please read the instructions below carefully and follow them meticulously. Start this homework as soon as you can and allow time for discussion and questions (slack!)._
 
-_Deadline Saturday_
+### 2.1 Introduction
 
-You are going to write a SPA (Single Page Application) that uses the [GitHub API](https://developer.github.com/guides/getting-started/).
+You are going to write a _Single Page Application_ (SPA) that uses the [GitHub API](https://developer.github.com/guides/getting-started/).
 
-This application should display information about the available [HYF repositories](https://github.com/hackyourfuture):
+This application should display information about the available [HYF GitHub repositories](https://github.com/hackyourfuture):
 
 - You should be able to select a repository from a list of available repositories.
```

Comment:
maybe we can add "functionalities" or "requirements" on the top of this list?

Reply:
Can you give an example of what you mean?

Reply:
yes, so here you give a first indication of the functionalities of what the students should build. So maybe we can add:

functionalities we would like to see in your application:
- this application....
- this application...
- etc
- etc

It is only about the line above the list of functionalities, indicating that this should be the "end" product

Reply:
Got it! Done.